importers/test/test_djornl_parser.py

```diff
@@ -72,7 +72,7 @@ def test_config(self):
             parser.config("bananas")
 
     def test_load_no_manifest(self):
-        """test loading when the manifest does not exist"""
```

Comment:
???

can you restore this comment? Looks like a merge mistake.

Reply:
done.